src/ansys/pytwin/evaluate/twin_model.py

```diff
@@ -1759,6 +1801,62 @@ def generate_points(self, rom_name: str, on_disk: bool = True, named_selection:
             msg += f"\n{str(e)}."
             self._raise_error(msg)
 
+    def get_tbrom_output_field(self, rom_name: str):
+        """
+        Return the TBROM output field from point cloud data.
```

Comment:
I think we should rework a bit the naming convention. "TBROM output field from point cloud data" does not make a lot of sense to me. I would suggest we introduce a new concept with a new class, called `field` that would contain the tbrom._pointsdata, as well as the tbrom.field_output_name (i.e. tbrom._outname).